FirebaseAnalyticsSwift.podspec

```
@@ -46,4 +46,15 @@ Firebase Analytics is a free, out-of-the-box analytics solution that inspires ac
       'FirebaseAnalyticsSwift/Tests/SwiftUnit/**/*.swift',
     ]
   end
+
+    s.test_spec 'objc-api-coverage' do |objc_api_tests|
```

nit: spacing so this test spec block is matches with line 39
```suggestion
  s.test_spec 'objc-api-coverage' do |objc_api_tests|
```